astropy/coordinates/solar_system.py

```diff
@@ -75,12 +76,12 @@
 
 
 class solar_system_ephemeris(ScienceState):
-    """Default ephemerides for calculating positions of Solar-System bodies.
+    f"""Default ephemerides for calculating positions of Solar-System bodies.
```

Comment:
This sadly does not work in python (I tried myself for another package!). You'll have to attach the docstring after the fact, like is already done with `EPHEMERIS_NOTE`

Reply:
Ah, thanks for catching, I didn't know that.